### PR TITLE
Cursed(?) Dealer Only Drawing Option

### PR DESCRIPTION
**Pull request by SonyUSA**
I think it works???

### CODE REVIEW

**Comment by SonyUSA** [comment]
Add player name to the Copy Player to more clearly distinguish when it is their turn, alter the formatting to be more eyeball friendly. Alter some text to be a little more friendly. Add the Totals to the GUI to take the load off the dealer from having to memorize/scroll back in chat.

**Comment by SonyUSA** [comment]
Turn some of the spaghetti into delicious ravioli per Infi <3

**Comment by SonyUSA** [comment]
Move some buttons around. Table player names can be clicked to copy their hand out any time.

**Comment by SonyUSA** [comment]
Copy Payout will let the Dealer paste a shortened (First name only) string with all payouts. This lets all Players see the final state of the game and give the Dealer a record to refer to when players want to move their bet (push) or winnings to the next round.

File: DeathRoll/Gui/Settings/BlackjackSettings.cs [hunk 1]
```diff
@@ -96,6 +96,19 @@ public void RenderSettings()
             Helper.ShowHelpMarker("Automatically draw all dealer cards (first two cards are excluded).");
         }
         
+        if (configuration.BlackjackMode == 1)
```
Comment (Infiziert90) [inline DeathRoll/Gui/Settings/BlackjackSettings.cs]: can be made a simple "else" as there is only 0 and 1 for blackjake mode

File: DeathRoll/Gui/BlackjackMode.cs [hunk 2]
```diff
@@ -130,48 +140,74 @@ public void MatchDonePanel()
...
         }
+
+        // Copy out all winnings to a single string to show all players the final standings (and leave a record if a player wants to play their push/winnings into the next round)
+        var finalPayout = $"Payouts: [] ";
+        if (ImGui.Button("Copy Payout"))
+        {
+            foreach (var (name, player) in participants.PlayerBets)
+            {
+                if (player.Bet < 1)
+                { finalPayout += $"{name.Split()[0]} -> Lost! [] "; }
```
Comment (Infiziert90) [inline DeathRoll/Gui/BlackjackMode.cs:151]: What are the "[]" for?

File: DeathRoll/Configuration.cs [hunk 3]
```diff
@@ -44,6 +44,7 @@ public class Configuration : IPluginConfiguration
     public bool AutoDrawCard = true;
     public bool AutoDrawOpening = true;
     public bool AutoDrawDealer = true;
+    public bool DealerDrawsAll = true;
```
Comment (Infiziert90) [inline DeathRoll/Configuration.cs]: We don't force new ways on the user, so we set it to false by default

**Review by Infiziert90 — CHANGES_REQUESTED** [review]

File: DeathRoll/Gui/BlackjackMode.cs (hunk 2 above)
Comment (SonyUSA) [reply DeathRoll/Gui/BlackjackMode.cs:151]: Makes it pretty :)

File: DeathRoll/Gui/Settings/BlackjackSettings.cs (hunk 1 above)
Comment (SonyUSA) [reply DeathRoll/Gui/Settings/BlackjackSettings.cs]: Fixed!

File: DeathRoll/Configuration.cs (hunk 3 above)
Comment (SonyUSA) [reply DeathRoll/Configuration.cs]: Fixed!

**Pull request merged**